scalafmt-tests/jvm/src/test/scala/org/scalafmt/ScalafmtProps.scala

```diff
@@ -19,12 +19,12 @@ class ScalafmtProps extends FunSuite with FormatAssertions {
   def getBugs(
       config: ScalafmtConfig = ScalafmtConfig.default,
       count: Int = Int.MaxValue,
-  ): mutable.Seq[(CorpusFile, Observation[Bug])] = {
+  ): mutable.Seq[(scala.meta.testkit.CorpusFile, Observation[Bug])] = {
```

Comment:
why is this package qualification necessary?

Reply:
Ah, sorry, leftover I have missed, removing now